website/static/js/addProjectPlugin.js

```diff
@@ -89,6 +96,15 @@ var AddProject = {
             var success = function _success (result) {
                 self.viewState('success');
                 self.goToProjectLink(result.data.links.html);
+                if (self.newProjectInstitutions.length > 0){
```

Comment:
Even when you create a project with no institutions selected, a request gets sent to the /relationships/institutions endpoint. I think you can avoid this with something like

```js
if (Object.keys(self.checkedInstitutions).length) 
```